README.md

```diff
@@ -230,6 +231,14 @@ Otherwise, the `callback` function will be called with the following 3 arguments
 
 
 --------------------------------------------------------
+<a name="iterator_seek"></a>
+### iterator#seek(key)
+<code>seek()</code> is an instance method on an existing iterator object, used to seek the underlying LevelDB iterator to a given key.
+
+by calling <code>seek(key)</code> subsequent calls to <code>next(cb)</code> will return key/values larger or smaller than <code>key</code>
```

Comment:
I suggest updating this to

By calling <code>seek(key)</code>, subsequent calls to <code>next(cb)</code> will return key/values larger or smaller than <code>key</code>, based on your <code>reverse</code> setting in the iterator constructor.